tests/bluechi_test/test.py

```diff
@@ -209,6 +211,7 @@ def run(self, exec: Callable[[BluechiControllerMachine, Dict[str, BluechiAgentMa
         try:
             self.shutdown_bluechi(ctrl_container, node_container)
         except Exception as ex:
+            test_result = ex
```

Comment:
Where is `test_result` being used?

Reply:
It's used to fail the test at https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/test.py#L228

Reply:
Ah, missed that. Should we also add this line to https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/test.py#L222?

Reply:
Usually not, but I am afraid that we might miss errors then or get rather cryptic errors later on. Both - adding it or keeping it as it is - is fine for me.

Reply:
OK, I will add it also there

Reply:
Added